openassessment/xblock/openassessmentblock.py

```diff
@@ -71,7 +71,7 @@
         "name": "staff-assessment",
         "class_id": "openassessment__staff-assessment",
         "navigation_text": "Staff assessment of your response",
```

Comment:
It looks navigation_text is unused in the code-- I will delete it.

And the title is currently not being translated (here is the place it is referenced-- https://github.com/edx/edx-ora2/blob/master/openassessment/templates/openassessmentblock/oa_base.html#L25). I will add translation in the template, as I don't have access to self._ in the UI_MODELS declaration. The titles should be scraped elsewhere because they should match the title shown once initialization is complete.